modules/machine-autoscaler-crd.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * control-plane-management/applying-autoscaling.adoc
+
+[id='machine-autoscaler-crd-{context}']
+= MachineAutoscaler resource definition
+
+This MachineAutoscaler resource definition shows the parameters and sample
+values for the MachineAutoscaler.
+
+
+[source,yaml]
+----
+apiVersion: "autoscaling.openshift.io/v1alpha1"
+kind: "MachineAutoscaler"
+metadata:
+  name: "worker-us-east-1a" <1>
+  namespace: "openshift-cluster-api"
+spec:
+  minReplicas: 1 <2>
+  maxReplicas: 12 <3>
+  scaleTargetRef:
+    apiVersion: machine.openshift.io/v1alpha1
+    kind: MachineSet
+    name: worker <4>
+----
+<1> Specify the `MachineAutoscaler` name, which is composed of the name of the
```

Comment:
This is fine, but maybe just a point of clarification. This is actually just the name of the particular MachineAutoscaler you are creating. It can be anything you want, but we should strongly encourage users to simply use the same name as the MachineSet being targeted for autoscaling.